config.m4

```diff
@@ -375,7 +375,11 @@ if test "$MONGODB" != "no"; then
 
   m4_include(src/libmongoc/build/autotools/m4/ax_prototype.m4)
   m4_include(src/libmongoc/build/autotools/CheckCompiler.m4)
+
+  dnl We need to convince the libmongoc M4 file to actually run these checks for us
+  enable_srv=auto
```

Comment:
Interesting. Is this necessary because we do not include [`ReadCommandLineArguments.m4`](https://github.com/mongodb/mongo-c-driver/blob/1.9.0-rc0/build/autotools/ReadCommandLineArguments.m4)? AFAIK, this is one of the few times we have to set a variable this way (instead of using AC replacement).

Reply:
Yup.